.github/workflows/base-package-tests.yaml

```
@@ -18,7 +18,7 @@ on:
 jobs:
   package-tests:
     name: Package Tests
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

we don't want to upgrade here, there's some issue with it
```suggestion
    runs-on: ubuntu-22.04
```